metriq_gym/run.py

```diff
@@ -61,9 +61,26 @@ def dispatch_job(args: argparse.Namespace, job_manager: JobManager) -> None:
     )
     logger.info(f"Job dispatched with ID: {job_id}")
 
-
 def poll_job(args: argparse.Namespace, job_manager: JobManager) -> None:
     logger.info("Polling job...")
+    if not args.job_id:
+        jobs = job_manager.get_jobs()
+        if not jobs:
+            print("No jobs available for polling.")
+            return
+        print("Available jobs:")
+        list_jobs(jobs, show_index=True)
+        while True:
+            try:
+                selected_index = int(input("Select a job index: "))
+                if 0 <= selected_index < len(jobs):
+                    break
+                else:
+                    print("Invalid index. Please enter a number between 0 and", len(jobs) - 1)
```

Comment:
nit: you want to use an f-string here, see https://realpython.com/python-f-strings/